package.json

```diff
@@ -26,9 +29,9 @@
 		"queries"
 	],
 	"dependencies": {
-		"gulp-util": "~2.2.14",
-		"through2": "~0.4.1",
-		"group-css-media-queries": "~1.0.0"
+    "group-css-media-queries": "~1.1.0",
```

Comment:
Can you please fix this whitespace issue? Have the indentation respect the previous format (tab).